### PR TITLE
add documentation for conic path not supported

### PR DESCRIPTION
**Pull request by cyanglaz**
## Description

Adding document to explain that iOS platform view clipping is not fully supported for conic path.

## Related Issues

https://github.com/flutter/flutter/issues/35062

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/widgets/platform_view.dart [hunk 1]
```diff
@@ -199,6 +200,8 @@ class AndroidView extends StatefulWidget {
 ///
 /// Construction of UIViews is done asynchronously, before the UIView is ready this widget paints
 /// nothing while maintaining the same layout constraints.
+/// 
+/// If a conic path clipping is applied to a UIKitView, a quad path is used to approximate the clip due to limitation of Quartz.
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/platform_view.dart]: nit: we try to keep doc comment lines to be ~80 characters long. This one locks longer.

**Review by amirh — COMMENTED** [review]
LGTM

File: packages/flutter/lib/src/widgets/platform_view.dart (hunk 1 above)
Comment (cyanglaz) [reply packages/flutter/lib/src/widgets/platform_view.dart]: It's definitely longer :), I will edit before merging.

File: packages/flutter/lib/src/widgets/platform_view.dart [hunk 2]
```diff
@@ -199,6 +200,8 @@ class AndroidView extends StatefulWidget {
 ///
 /// Construction of UIViews is done asynchronously, before the UIView is ready this widget paints
 /// nothing while maintaining the same layout constraints.
+///
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/platform_view.dart]: looks like there's a whitespace at the end of this line that makes cirrus mad. :)

Comment (cyanglaz) [reply packages/flutter/lib/src/widgets/platform_view.dart]: Good catch! Thanks. VSCode always generates `/// ` with an empty space in the end when hit return key on the keyboard after a line of dart doc.

**Pull request merged**